pages/news/2020-07-08-intro-outro-sections.html

```diff
@@ -0,0 +1,106 @@
+
+## Intro & Outro Cues With AutoDJ ##
+
+In Mixxx 2.3, these new intro & outro section cues can be used to tell AutoDJ how long to crossfade and how to align the tracks in time.
+AutoDJ uses the intro & outro cues in the new "Full Intro + Outro" and "Fade At Outro Start" modes.
+In both modes, instead of crossfading over an arbitrary number of seconds, AutoDJ compares the duration of the outro of the old track and the intro of the new track.
+AutoDJ picks the time of the shorter section as the crossfade time.
+
+If the outro is shorter than the intro, AutoDJ will align the start of the outro with the start of the intro in both modes.
+
+![AutoDJ aligning the start of the intro and outro](/static/images/news/intro-outro/autodj-align-starts.png)
+
+The two modes work differently when the outro is longer than the intro.
+In the "Full Intro + Outro" mode, AutoDJ aligns the end of the intro and outro by starting the next track during the outro of the previous track.
+This way, the full length of both the intro and the outro are played.
+This mode is the most likely to sound good with the widest variety of tracks.
+Therefore, it is the default mode.
+
+![AutoDJ aligning the end of the intro and outro in Full Intro + Outro mode](/static/images/news/intro-outro/autodj-align-ends.png)
+
+The "Fade At Outro Start" mode always aligns the start of the intro and outro.
+When the outro is longer than the intro, AutoDJ cuts off the end of the outro.
+This can be helpful if you want to prevent the energy of the mix from declining during a long outro. However, the transition may sound abrupt if the intro is short.
+
+![AutoDJ aligning the start of the intro and outro in Fade At Outro Start mode](/static/images/news/intro-outro/autodj-fade-at-outro-start.png)
+
+Both modes were designed so that AutoDJ sounds good fading between any pair of tracks if the intro and outro sections have been marked reasonably.
+AutoDJ does not take into account the volume of each track, nor the frequency content, nor the rhythms, so it's not intended to be a replacement for a human DJ.
+However, it is good enough to give a human DJ a break without a major disruption to the mix.
+
+If you still want the old behavior of AutoDJ using a fixed number of seconds to crossfade, that is still available with the "Full Track" mode.
+The new "Skip Silence" mode behaves the same way, but cuts out the silence at the beginning and end of tracks.
+
+## Section Detection ##
+
+Now that we know how useful it is to mark intro & outro sections, we are planning to take this concept further. As part of our [Google Summer of Code 2020 projects](/news/2020-05-05-mixxx-gsoc-projects-2020), we'll be adding downbeat, phrase, and section markers to Mixxx and extending the analyzers to detect these.
```

Comment:
We could make it more personal and name the students:
```suggestion
Now that we know how useful it is to mark intro & outro sections, we are planning to take this concept further. As part of our [Google Summer of Code 2020 projects](/news/2020-05-05-mixxx-gsoc-projects-2020), our GSoC students Harshit and Cristiano will work on adding downbeat, phrase, and section markers to Mixxx and extending the analyzers to detect these.
```